pyrelational/data/data_manager.py

```diff
@@ -10,6 +25,10 @@
 logger = logging.getLogger()
 
 
+class SizedDataset(Dataset, Sized):
```

Comment:
now that I see it here, I am thinking we should move it to a types.py file somewhere. doesn't feel right :p

Reply:
done :)